src/holoscan_cli/common/constants.py

```diff
@@ -77,19 +77,22 @@ class SDK:
     """
 
     # Platform to architecture mappings
-    PLATFORM_MAPPINGS = {
-        Platform.IGXOrinDevIt: Arch.arm64,
-        Platform.JetsonAgxOrinDevKit: Arch.arm64,
+    PLATFORM_ARCH_MAPPINGS = {
+        Platform.Jetson: Arch.arm64,
+        Platform.IGX_iGPU: Arch.arm64,
+        Platform.IGX_dGPU: Arch.arm64,
         Platform.SBSA: Arch.arm64,
         Platform.X64Workstation: Arch.amd64,
```

Comment:
Remove X64Workstation? or is it still needed for backwards compatibility? How far back?

Reply:
Thanks. Removed